### PR TITLE
Fix/node7

### PR DESCRIPTION
**Pull request by nfroidure**

### CODE REVIEW

**Comment by SebastienElet** [comment]
Cool 👍

**Comment by nfroidure** [comment]
Looks like Node7 is printing stack traces at Error creation. It needed a few arrangements to work. YHTTPError will also need an upgrade.

Beware that this breaks up Node 4 retro compatibility.

**Comment by SebastienElet** [comment]
I seen, this will need a major bump.

Node 4 will be in maintenance in 2 months and this version will let me upgrade to node7 !

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "preversion": "npm run lint && npm test"
   },
   "engines": {
-    "node": ">= 4"
+    "node": ">=6.9.5"
```
Comment (SebastienElet) [inline package.json]: Why older release `6.x.x` are not supported ?

Comment (nfroidure) [inline package.json]: Just fitting the recent LTS. Feel free to rearrange it.

**Comment by nfroidure** [comment]
You do not have to wait that moment since users of Node 4 version will just have to avoid using the new major version. I need a Node7 compatible version to fix my builds on this target.

**Comment by SebastienElet** [comment]
I look to make it work with older node 6 release and merge it after.

Travis use 6.9.5 also for the node 6 build :/

**Comment by nfroidure** [comment]
Yes, i think it is expected, you have to manually specify which lower node version you want to be compatible with.

In my case, i always support at least the last Node LTS version only: https://github.com/nfroidure/siso/blob/master/.travis.yml#L7-L8

Note that it doesn't impeach anyone to install it on older NodeJS version but they may face issues.

**Pull request merged**